### DISCUSSION TITLE
Class 'REST_Controller' not found

### DISCUSSION THREAD

**Discussion by bdtaskrony** [madeindra/codeigniter-microservice#10] [Q&A]
When i setup this project and execute this (index.php/api/v1/invoices) API it shows this error
https://prnt.sc/1y4jzo8
and I also can't find the Rest_Controller in the whole project.

**Comment by madeindra** [comment]
@bdtaskrony Have you followed the steps to install dependency?

**Reply by bdtaskrony** [reply]
I did not install docker and I used it without docker in the local apache server. But in the whole file, I can not find the Rest_Controller. Please mention from where you extend this Rest_Controller?
https://prnt.sc/1y5570a

**Reply by madeindra** [reply]
Pardon me, but it's installing dependency, not docker.

This part, have you done it? Please make sure you have `composer` installed before running these commands.

## Install dependency using script (recommended)
```
chmod +x ./install-dependency.sh
./install-dependency.sh
```

## Install dependency manually (If you are unable to run the script)
```
cd financial
composer install
cd ..
cd order
composer install
cd ..
cd warehouse
composer install
cd ..
```

**Reply by madeindra** [reply]
The `REST_Controller` came from this dependency
https://github.com/chriskacerguis/codeigniter-restserver
